react/features/prejoin/actions.js

```diff
@@ -58,6 +99,124 @@ export function addPrejoinContentSharingTrack(value: Object) {
     };
 }
 
+/**
+ * Polls for status change after dial out.
+ * Changes dialog message based on response, closes the dialog if there is an error,
+ * joins the meeting when CONNECTED.
+ *
+ * @param {string} reqId - The request id used to correlate the dial out request with this one.
+ * @param {Function} onSuccess - Success handler.
+ * @param {Function} onFail - Fail handler.
+ * @param {number} count - The number of retried calls. When it hits STATUS_REQ_CAP it should no longer make requests.
+ * @returns {Function}
+ */
+function pollForStatus(
+        reqId: string,
+        onSuccess: Function,
+        onFail: Function,
+        count = 0) {
+    return async function(dispatch: Function, getState: Function) {
+        try {
+            if (!isJoinByPhoneDialogVisible(getState())) {
```

Comment:
store getState in a variable, since it's ussed in 2 ssubsequent lines?

Reply:
Indeed. Thanks!